lrauv_ignition_plugins/test/test_mission_pitch_and_depth_mass_vbs.cc

```diff
@@ -88,19 +88,19 @@ TEST_F(LrauvTestFixture, PitchDepthVBS)
   for (const auto pose: this->tethysPoses)
   {
     // Vehicle should dive down.
-    EXPECT_LT(pose.Pos().Z(), 0.1);
+    EXPECT_LE(pose.Pos().Z(), 0.5);
```

Comment:
This makes sense as there is minor oscillation.

Reply:
It's not the oscillation, but the fluid density changing at z = 0.5m. I was meaning to ask, should the interface be at -0.5m instead?